.chloggen/exporter-splunkhec-addbatcherconfig.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: exporter/splunkhec
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: add exporter batcher config in splunk hec exporter
```

```suggestion
note: add exporter batcher config
```
Nit: In the final release notes the component name will prefix this note, so the component will be obvious from that.